soil_params_Gupta2020_2022/utils.jl

```diff
@@ -35,24 +35,25 @@ function regrid(data, (lon, lat), resolution, transform, nlayers)
             # Here we make a land mask by checking where x is *not* Missing.
             x_land_mask = .!ismissing.(x)
             if sum(x_land_mask) / prod(size(x)) > 0.5 # count as land
-                outdata[lon_id, lat_id, :] .= transform(mean(x[x_land_mask]))
+                m = [mean(skipmissing(x[:, :, k])) for k in 1:nlayers]
```

Comment:
looks good! I had to think about this for a second, but I think this is right - we want to take the mean first (I think in log-space for alpha and K_sat), and then transform to linear space.